File: launchdarkly/php/src/GetVersionsExample.php
<?php

namespace OSEG\LaunchDarklyExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use LaunchDarkly;

$config = LaunchDarkly\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("Authorization", "YOUR_API_KEY");

try {
    $response = (new LaunchDarkly\Client\Api\OtherApi(config: $config))->getVersions();

    print_r($response);
} catch (LaunchDarkly\Client\ApiException $e) {
    echo "Exception when calling OtherApi#getVersions: {$e->getMessage()}";
}

File: launchdarkly/ruby/src/GetVersionsExample.rb
require "json"
require "launchdarkly_client"

LaunchDarklyClient.configure do |config|
    config.api_key["Authorization"] = "YOUR_API_KEY"
end

begin
    response = LaunchDarklyClient::OtherApi.new.get_versions

    p response
rescue LaunchDarklyClient::ApiError => e
    puts "Exception when calling OtherApi#get_versions: #{e}"
end
